src/wp-includes/css/admin-bar.css

```diff
@@ -1016,6 +1052,25 @@ html:lang(he-il) .rtl #wpadminbar * {
 		top: 10px;
 		left: 0;
 	}
+	/* Prevent wrapping of admin bar that has more items than admin bar area on mobile layout */
+	#wpcontent {
+		padding-top: 15px;
+	}
+	#wpadminbar {
+		display: flex;
+		flex-wrap: nowrap;
+		padding-bottom: 15px;
```

Comment:
15px beign the browser scrollbar dimension per https://codepen.io/sambible/post/browser-scrollbar-widths (with the overflow-y:hidden; handling the difference between them)